backend/src/users/controllers.ts

```diff
@@ -178,6 +178,33 @@ const getAllUsers = async (req: Request, res: Response) => {
     res.status(500).json({ error: (error as Error).message });
   }
 };
+
+/**
+ * Gets all Users in database with pagination
+ * @returns promise with all users or error
+ *
+ *
+ */
+const getUsersPaginated = async (req: Request, res: Response) => {
+  // #swagger.tags = ['Users']
+  try {
+    const query = req.query;
+    const users = await prisma.user.findMany({
+      take: query.limit ? parseInt(query.limit as string) : 10,
+      cursor: {
+        id: query.after ? query.after as string : undefined,
+      },
+      orderBy: {
+        id: "asc",
+      },
+    }
+    );
+    res.status(200).json(users);
+  } catch (error) {
+    res.status(500).json({ error: (error as Error).message });
```

Comment:
You don't have to do this but ideally we should have a `utils/` folder with success and error json methods to standardize error responses